### PR TITLE
Allow pushing images from a fork

### PR DESCRIPTION
**Pull request by tomwilkie**
Before you merge this PR, you'll need to set the Circle CI env var `IMAGE_PREFIX=quay.io/weaveworks/cortex-`.

Otherwise everything else should just work.

### CODE REVIEW

**Comment by tomwilkie** [comment]
Of course, no worries!

File: circle.yml [hunk 1]
```diff
@@ -25,5 +25,5 @@ deployment:
     branch: master
     commands:
       - docker login -e "$DOCKER_REGISTRY_EMAIL" -u "$DOCKER_REGISTRY_USER" -p "$DOCKER_REGISTRY_PASSWORD"
-      - docker login -e '.' -u "$QUAY_USER" -p "$QUAY_PASSWORD" quay.io
-      - ./push-images
+      - if [ -n "$QUAY_PASSWORD" ]; then docker login -e '.' -u "$QUAY_USER" -p "$QUAY_PASSWORD" quay.io; fi
+      - ./push-images "$NOQUAY"
```
Comment (bboreham) [inline circle.yml]: Where is `$NOQUAY` set?  Did you mean to call with a blank string if not set? (Looks like this would lead to 'usage' message)

Comment (jml) [inline circle.yml]: It does indeed lead to said message.  See #498, which reverts this PR.

Comment (tomwilkie) [inline circle.yml]: Good catch, I didn't mean to call with an empty string.  Will update and open a new PR.

**Pull request merged**